problems/arraysWithLoops.js

```diff
@@ -66,24 +111,38 @@ function findSmallest() {}
  * @returns {number} second smallest value
  */
 
-function findSecondSmallest() {}
+const findSecondSmallest = (nums) => {
+  let secondSmallest = Math.min.apply(null, nums);
+  nums.indexOf(secondSmallest), 1
+  return secondSmallest +1;
+}
 
 /**
  * Takes in array and returns the second largest number in the array
  * @param {number[]} nums
  * @returns {number} second largest value
  */
 
-function findSecondLargest() {}
-
+const findSecondLargest = (nums) => {
+let secondLargest = Math.max.apply(null, nums);
+nums.indexOf(secondLargest), 1
+return secondLargest -1;
+}
 /**
  * Takes in array and returns an array with all the values but with no duplicates.
  * Exp Input: [4,2,6,2,2,6,4,9,2,1] Output: [4, 2, 6, 9, 1]
  * @param {number[]} nums
  * @returns {array} nums without the duplicates
  */
 // Hint: Look into the `.includes` method.
-function removeDups() {}
+const removeDups = (nums) => {
+  let array = [... new Set(nums)];{
```

Comment:
watch you syntax. There should not be {} after the line on 139
How would you do this without using set?